packages/config/config/devices/0x0005/pe653.json

```
@@ -55,5 +55,63 @@
 			"maxValue": 7,
 			"defaultValue": 0
 		}
+	],
+	"endpoints": {
+		"0": {
+			"associations": {
+				"1": {
+					"label": "Lifeline",
+					"maxNodes": 5,
+					"isLifeline": true
+				}
+			}
+		}
+	},
```

This isn't 100% clear from the documentation, but the default is endpoint 0
```suggestion
	"associations": {
		"1": {
			"label": "Lifeline",
			"maxNodes": 5,
			"isLifeline": true
		}
	},
```